networkx/algorithms/mst.py

```diff
@@ -21,6 +22,33 @@
 from itertools import count
 
 
+def _optimum_spanning_edges(G, min, weight='weight', data=True):
```

Comment:
I like @jfinkels suggestion that this should be something like `G, minimum=True, weight='weight'`. This will also make your code later simpler.